cloudslang-runtime/src/main/resources/scripts/cs_append.py

```diff
@@ -0,0 +1,2 @@
+def cs_append(value, to_append):
+  return value + to_append
```

Comment:
This works only for strings. If the user explicitly casts the arguments to int, it's an addition

Reply:
Nice catch @alexbajzat, I believe we need to force value to string.